platform/search/vector-database.html

```diff
@@ -0,0 +1,115 @@
+---
+layout: fullwidth
+title: Using OpenSearch as a Vector Database
+primary_title: Using OpenSearch as a Vector Database
+---
+<div class="platform-solution">
+    <hgroup>
+        <h2>An open-source, all-in-one vector database for building flexible, scalable, and future-proof AI applications</h2>
+        <div class="platform-solution--body">
+            <p>
+                Traditional term-based search, based on term frequency models like BM25, is widely used and effective for many search applications. However, these term-based search techniques require significant investment in time and expertise to tune them to account for the meaning or relevance of the terms searched.  Today, more and more developers want to embed semantic understanding into their search applications. Enter artificial intelligence (AI) embedding models that can encode the meaning and context of documents, images, and audio into vectors for similarity search. These embedded meanings can, in turn, be searched using the <a href="https://opensearch.org/docs/latest/search-plugins/knn/index/">k-nearest neighbors (k-NN)</a> functionality provided by OpenSearch.  
```

Comment:
> 
Let's address by changing phrasing from "Enter artificial intelligence (AI) embedding models" to "Enter machine learning embedding models". No need to say (ML) after machine learning because term is not used again on page at this time.

Reply:
> Additionally, we should call out approximate nearest neighbor (ANN) as it is a feature that not all other vector DB's support and is how people can improve the latency and scale of kNN search.

Suggest this and other suggestions for changes to page made today (aside from AI -> ML above) be tracked for future round of updates to this page. @dtaivpp  does that work for you?

Reply:
Requested changes have been pushed.